Corona-Warn-App/src/main/res/layout/bugreporting_upload_history_fragment.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <Toolbar
+        android:id="@+id/toolbar"
+        style="@style/CWAToolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:navigationIcon="@drawable/ic_back"
+        android:title="@string/debugging_debuglog_uploadhistory_title" />
+
+    <androidx.core.widget.NestedScrollView
```

Comment:
Can't test this atm, but since we got some performance issue when using `NestedScrollView` in the `ContactDiaryDayFragment` on some devices, this may could also affect this screen when someone got a lot of entries. 
Maybe we can remove `NestedScrollView` and add description as part of the recycler view